llvm/test/CodeGen/SystemZ/scmp.ll

```diff
@@ -0,0 +1,130 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=s390x-linux-gnu -mcpu=z13 | FileCheck %s
+
+define i8 @scmp.8.8(i8 %x, i8 %y) nounwind {
```

Comment:
Here also signext/zeroext.

It also looks like the tests here regressed with the new expansion. Should probably implement the TLI hook for s390x as well?

Reply:
Thanks for spotting this, I completely missed it yesterday.